### ISSUE TITLE
tlg0057.tlg001-two editions of same work-different titles

### ISSUE THREAD

**Issue by AlisonBabeu** [OpenGreekAndLatin/First1KGreek#796]
@sonofmun , I was wondering why tlg0057.tlg001.1st1k-grc1 has the work title-Adhortatio ad artes addiscendas while tlg0057.tlg001.1st1k-grc2 has the title Protepticus. I know that both titles are used for this work but was wondering if we should pick one, or is the practice going to be to use a work title as found in the edition and let the ID speak for itself?

**Comment by lcerrato** [comment]
This is not uncommon with Perseus texts. 
I have been adding alternate titles to the CTS work file.
In most cases, I try to keep the xml file title as close to the print edition as possible, in keeping with the editing practices.

**Comment by sonofmun** [comment]
I agree with Lisa here. Name the work what the edition names it. The `__cts__.xml` file will keep track of the accepted name for the work (as well as the differing titles for any other edition).
